### DISCUSSION TITLE
Possibility to upload record videos

### DISCUSSION THREAD

**Discussion by trooperjey01** [DamianOsipiuk/testcafe-reporter-testrail#26] [Ideas]
Hi, 

I used the record videos option of Testcafe for my runs.   

Like upload screenshot is it possible to add an option TESTRAIL_UPLOAD_VIDEO to upload these records on TestRail?

**Comment by DamianOsipiuk** [comment]
Hi, it is possible.
But you will have to watch for the space usage as videos might be very heavy. There is also a limit for a single attachment. Something around 200 MB.

I can add that option, but unfortunately I do not have access to any Testrail instance anymore, so I won't be able to test the integration fully.
It should work though since it should be very similar to screenshot upload.

**Reply by trooperjey01** [reply]
Hi, 

Thanks for your answer.

My records are not so big so it won't be a problem. 

I have been testing with records recently with few tests so I've uploaded them manually but I need to expand this feature with my all test cases. It will be very helpful if you add this option. 

If you want I can test the option and give you feedback.

**Reply by DamianOsipiuk** [reply]
@trooperjey01 You could try https://github.com/DamianOsipiuk/testcafe-reporter-testrail/releases/tag/v0.7.0 and let me know.
It's not fully tested as I have currently no access to any TestRail instance.

I hope I did not break anything along the way :crossed_fingers:

**Reply by trooperjey01** [reply]
@DamianOsipiuk I've just tested with 3 test cases and it looks good :-). Screenshots and records are uploaded in TestRail. 

Thanks a lot!
